scenario/tasks.yaml

```diff
@@ -68,3 +68,5 @@ tasks:
    check_needed_compute_resources: ['CheckNeededComputeResources']
    act_check_bandwidth_src: ['CheckBandwidth', 'src_cloud']
    act_check_bandwidth_dst: ['CheckBandwidth', 'dst_cloud']
+   get_volumes_db_data: ["GetVolumesDb", 'src_cloud']
+   write_volumes_db_data: ["WriteVolumesDb", 'dst_cloud']
```

Comment:
Should we create scenario specific to our goal based on the new tasks we added? This would be very helpful for the test runs.